src/ui/public/stringify/register.js

```diff
@@ -12,6 +13,7 @@ import stringifyTruncate from 'ui/stringify/types/truncate';
 fieldFormats.register(stringifyUrl);
 fieldFormats.register(stringifyBytes);
 fieldFormats.register(stringifyDate);
+fieldFormats.register(stringifyDuration);
```

Comment:
Not for the OP, but we really need to make this list pluggable so we don't need this file.